source/manual/get-started.html.md

```diff
@@ -185,13 +185,7 @@ There are also some Python apps, which use [pip][]. You’ll probably need to in
 
     dev$ ./update-pip.sh
 
-If installing the Python dependencies for fabric-scripts fails, your version of setuptools may be too old:
-
-    dev$ cd /var/govuk/fabric-scripts
-    dev$ virtualenv .venv
-    dev$ source .venv/bin/activate
-    dev$ pip install --upgrade setuptools
-    dev$ pip install -r requirements.txt
+If installing the Python dependencies fails, try and `cd` into each failing repository (e.g. `cd /var/govuk/fabric-scripts`) and remove the venv file (`rm -rf venv`) before running the script again.
```

Comment:
You're missing a dot from `.venv` (it's a hidden directory)

Reply:
Thanks Bruce!